version.php

```diff
@@ -28,7 +28,7 @@
 defined('MOODLE_INTERNAL') || die();
 
 $plugin->component    = 'mod_studentquiz';
-$plugin->version      = 2022020200.02;
+$plugin->version      = 2022080300.08;
```

Comment:
Note, Moodle plugin database does not like decimal points in plugin version numbers. Please change this to be

2022080308 instead of 2022080300.08

that is what those last two digits of the vertsion number are for.

(The use of decimal points in Moodle core is for moodle core top-level version.php only.)